ExtIO_sddc/r2iq.cpp

```diff
@@ -92,9 +92,9 @@ int r2iqControlClass::Setdecimate(int dec)
 	return mratio[mdecimation];
 }
 
-void r2iqControlClass::Updt_SR_LO_TUNE(int srate_idx, int64_t* fLO, int64_t* fTune)
+void r2iqControlClass::Updt_SR_LO_TUNE(int srate_idx, int64_t* pfLO, int64_t* pfTune)
```

Comment:
this function and UptTunerFrq is so similar. can we merge them?

Reply:
actually, I think this function should not belong to r2iq. All r2iq needs is just a tunebin offset. Maybe this logic should be in RadioHandler and set tubebin to r2iq.

Reply:
Yes your right Todo

Reply:
> this function and UptTunerFrq is so similar. can we merge them?
Done. Only UptTunerFrq now.